README.md

```diff
@@ -1,15 +1,24 @@
 # URLDownloader Service
+
 Assemblyline service that downloads seemingly malicious URLs using MAS' Kangooroo utility
 
-# Kubernetes VS Docker deployment
-In Kubernetes, there is a chance that you do not need to configure the no_sandbox option. If you are executing URLDownloader in a docker-compose setup, and have problem with it always finishing with an error (TimeoutExpired), you can change the "no_sandbox" service variable from the default False to True. This option will be passed on to the google-chrome process and may resolve your issue.
+## URLDownloader Service Configuration
+
+Administrators can find the configuration page for URLDownloader by going to the **Adiministration > Services** page, and click on **URLDownloader** (or by going to `/admin/services/URLDownloader` directly).
+_Service Variables_ and _User Specified Parameters_ can be found in the **Parameters** tab.
+
+More information on service management can be found in our documentation and more specifically [here](https://cybercentrecanada.github.io/assemblyline4_docs/administration/service_management/#service-variables) for service variables.
+
+<!-- # Kubernetes VS Docker deployment
```

Comment:
We shouldn't keep deprecated content in comment. It's all available in the git history.
After removing this, we should update this documentation specific for URLDownloader:
https://cybercentrecanada.github.io/assemblyline4_docs/installation/appliance/docker/#service-specific-configurations